project_name/settings/local_base.py

```diff
@@ -33,6 +33,27 @@
 EMAIL_BACKEND = 'naomi.mail.backends.naomi.NaomiBackend'
 EMAIL_FILE_PATH = base_dir_join('tmp_email')
 
+# django-debug-toolbar and django-debug-toolbar-request-history
+INSTALLED_APPS += ('debug_toolbar',)
+MIDDLEWARE += ('debug_toolbar.middleware.DebugToolbarMiddleware',)
+INTERNAL_IPS = '127.0.0.1'
```

Comment:
`INTERNAL_IPS` should be a list. I think we should follow a pattern like the one described in this [example settings](https://github.com/jazzband/django-debug-toolbar/blob/e7a9ed0c3de1c8635f7e8c83349cb77cef42489f/example/settings.py#L14).

Reply:
@rsarai any updates on this?

Reply:
Yeah! Sorry for the delay, you have this done this week!

Reply:
@chocoelho I see you've assigned yourself on Oct 20th, but shouldn't this be assigned to @rsarai ? :p